openmc/mesh.py

```diff
@@ -854,4 +854,505 @@ def from_xml_element(cls, elem):
         filename = get_text(elem, 'filename')
         library = get_text(elem, 'library')
 
-        return cls(filename, library, mesh_id)
+        mesh = cls(filename, mesh_id)
+
+        return mesh
+
+  
+class WeightWindowMesh:
+    """A three dimension Weight Window Mesh class
```

Comment:
Might want to say "dimensional" and not uppercase here, unless you're trying to do CamelCase, and if that is so, remove the spaces between the words.